taf/updater/updater.py

```diff
@@ -541,9 +545,21 @@ def _update_named_repository(
                         )
                         if error:
                             raise error
+                        return error
```

Comment:
This can only return `None` since it will only be returned if the error is not raised